modules/get-started/pages/whats-new.adoc

```diff
@@ -1,79 +1,95 @@
+- `rpk cluster partitions unsafe-recover`
+- `rpk cluster txn` (?)
 
-- xref:reference:rpk/rpk-profile/rpk-profile.adoc[`rpk profile`]: This allows users to switch between clusters seamlessly.
-- xref:reference:rpk/rpk-cloud/rpk-cloud-login.adoc[`rpk cloud login`]: This allows users to connect to cloud environments using a browser.
-- xref:reference:rpk/rpk-topic/rpk-topic-trim-prefix.adoc[`rpk topic trim-prefix`]: This allows users to delete records from a topic.
-- xref:reference:rpk/rpk-topic/rpk-topic-describe-storage.adoc[`rpk topic describe-storage`]: This provides detailed information about storage, including local and object disk usage and time of last write to object storage.
-- xref:reference:cluster-properties.adoc#legacy_permit_unsafe_log_operations[`legacy_permit_unsafe_log_operations`] and xref:reference:cluster-properties.adoc#legacy_unsafe_log_warning_interval_sec[`legacy_unsafe_log_warning_interval_sec`]: These properties enable a Redpanda cluster operator to use unsafe control characters within strings, such as consumer group names or user names.
+The following commands have new flags:
 
-== Documentation improvements
+- xref:reference:rpk/rpk-acl/rpk-acl.adoc[`rpk acl`]
+- xref:reference:rpk/rpk-container/rpk-container-start.adoc[`rpk container start`]
+- xref:reference:rpk/rpk-container/rpk-container-start.adoc[`rpk profile create`]
+- xref:reference:rpk/rpk-topic/rpk-topic-consume.adoc[`rpk topic consume`]
+- xref:reference:rpk/rpk-topic/rpk-topic-produce.adoc[`rpk topic produce`]
 
-Although not specific to version 23.2, the following sections are new or have been significantly enhanced in the documentation:
+The following commands have improved documentation or examples:
 
-- xref:api:ROOT:admin-api.adoc[Admin API]
-- xref:reference:topic-properties.adoc[Topic Configuration Properties]
-- xref:./intro-to-rpk.adoc[Redpanda CLI]
-- xref:develop:benchmark.adoc[Benchmark Redpanda]
-- xref:deploy:deployment-option/self-hosted/manual/production/production-deployment-automation.adoc[Automated Deployment]
-- xref:develop:config-topics.adoc[Manage Topics]
-- xref:manage:cluster-maintenance/decommission-brokers.adoc[Decommission Brokers]
+- xref:reference:rpk/rpk-cluster/rpk-cluster-config-set.adoc[`rpk cluster config set`]
+- xref:reference:rpk/rpk-cluster/rpk-cluster-partitions-move-status.adoc[`rpk cluster partitions move-status`]
+- xref:reference:rpk/rpk-debug/rpk-debug-bundle.adoc[`rpk debug bundle`]
+- xref:reference:rpk/rpk-redpanda/rpk-redpanda-config-set.adoc[`rpk redpanda config set`]
+- xref:reference:rpk/rpk-version.adoc[`rpk version`] 
+
+The xref:reference:rpk/rpk-cluster/rpk-cluster-storage.adoc[`rpk cluster storage`] command was renamed. It still work with aliases. 
+
+== Deprecated features 
+
+The `delete_retention_ms` property has been xref:upgrade:deprecated/index.adoc[deprecated]. Use `log_retention_ms` instead. 
```

Comment:
@mattschumpert should we report that `rpk cluster storage recovery` is deprecated as well, even though it still works as an alias for now.

Reply:
YES